README.md

```diff
@@ -5,7 +5,82 @@
 grmtools is a suite of Rust libraries and binaries for parsing text, both at
 compile-time, and run-time. Most users will probably be interested in the
 compile-time Yacc feature, which allows traditional `.y` files to be used
-(mostly) unchanged in Rust. See the
-[grmtools book](https://softdevteam.github.io/grmtools/master/book/)
-for more information, including a
-[quickstart guide](https://softdevteam.github.io/grmtools/master/book/quickstart.html).
+(mostly) unchanged in Rust.
+
+## Quickstart
+
+A minimal example using this library consists of two files (in addition to the
+grammar and lexing definitions). First we need to create a file `build.rs` in
+the root of our project with the following content:
+```
```

Comment:
Needs a blank line before the triple quotes and the triple quotes should be "```rust" to help with syntax highlighting (for those clients that know how to syntax highlight Rust).

Reply:
Ah, the syntax highlighting is fixed in a later commit (not quite sure why?!) but the blank line, I think, isn't.

Reply:
I fixed the newline as well in that commit.

Reply:
Ah OK.